user/src/components/header/Header.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import InputSearch from '../ui/input/input-search/InputSearch';
+import Navbar from '../ui/navbar/Navbar';
+import { navLefts, navRights } from './data.js'
+import './Header.scss'
+
+const Header = () => {
+    return (
+        <header className="header fixed-top shadow">
+            <div className='header-wrapper'>
+                <div className='header-top'>
+                    <div className='container'>
+                        <div className='header-top-wrapper row'>
+                            <div className='header__icon col-auto'>
+                                <a href='https://twitter.com/?lang=vi'>
+                                    <i className="fab fa-twitter"></i>
+                                </a>
+                                <a href='https://twitter.com/?lang=vi'>
+                                    <i className="fab fa-facebook-f"></i>
+                                </a>
+                                <a href='https://twitter.com/?lang=vi'>
+                                    <i className="fab fa-vimeo-v"></i>
+                                </a>
+                                <a href='https://twitter.com/?lang=vi'>
+                                    <i className="fab fa-youtube"></i>
+                                </a>
+                                <a href='https://twitter.com/?lang=vi'>
+                                    <i className="fab fa-pinterest-p"></i>
+                                </a>
+                            </div>
+                            <div className='header__search col-4 d-none d-md-block'>
+                                <InputSearch />
+                            </div>
+                            <div className='header__right col-auto'>
+                                <button className='button button-base'>Shop Sale</button>
+                                <div className='language px-3'>
+                                    <span>VIE</span>
+                                    <i className="fas fa-angle-down ms-1"></i>
+                                    <ul className='language__option shadow'>
+                                        <li className='language__item'>
+                                            VIE
+                                        </li>
+                                        <li className='language__item'>
+                                            ENG
+                                        </li>
```

Comment:
```suggestion
                                        <li className='language__item'>
                                            VI
                                        </li>
                                        <li className='language__item'>
                                            EN
                                        </li>
```
Ngôn ngữ nó sẽ lấy 2 chữ cái đầu thôi nhé